sdk/web-pubsub/web-pubsub-express/src/webPubSubEventHandler.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import express from "express";
+
+import { CloudEventsDispatcher } from "./cloudEventsDispatcher";
+import { WebPubSubEventHandlerOptions } from "./cloudEventsProtocols";
+
+/**
+ * The handler to handle incoming CloudEvents messages
+ */
+export class WebPubSubEventHandler {
+  /**
+   * The path this CloudEvents handler listens to
+   */
+  public readonly path: string;
+
+  private _cloudEventsHandler: CloudEventsDispatcher;
+
+  /**
+   * Creates an instance of a WebPubSubEventHandler for handling incoming CloudEvents messages.
+   *
+   * Example usage:
+   * ```ts
+   * import express from "express";
+   * import { WebPubSubEventHandler } from "@azure/web-pubsub-express";
+   * const endpoint = "https://xxxx.webpubsubdev.azure.com"
+   * const handler = new WebPubSubEventHandler('chat', [ endpoint ] {
+   *   handleConnect: async (req, res) => {
+   *     console.log(JSON.stringify(req));
+   *     return {};
+   *   },
+   *   onConnected: async req => {
+   *     console.log(JSON.stringify(req));
+   *   },
+   *   handleUserEvent: async (req, res) => {
+   *     console.log(JSON.stringify(req));
+   *     res.success("Hey " + req.data, req.dataType);
+   *    };
+   *  },
+   * });
+   * ```
+   *
+   * @param hub The name of the hub to listen to
+   * @param allowedEndpoints The allowed endpoints for the incoming CloudEvents request
+   * @param options Options to configure the event handler
+   */
+  constructor(
+    private hub: string,
+    allowedEndpoints: string[],
+    options?: WebPubSubEventHandlerOptions
+  ) {
+    const path = options?.path ?? `/api/webpubsub/hubs/${hub}`;
```

Comment:
Nit: since we're just going to have to add a forward slash in the next line anyway...
```suggestion
    const path = options?.path ?? `/api/webpubsub/hubs/${hub}/`;
```

Reply:
nice catch